File: Python/URI 1013.py
linha = input().split()
num1, num2, num3 = linha

if int(num1) >= int(num2) and int(num1) >= int(num3):
    print("{} eh o maior".format(num1))
elif int(num2) >= int(num1) and int(num2) >= int(num3):
    print("{} eh o maior".format(num2))
else:
    print("{} eh o maior".format(num3))

File: Java/URI 1013.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      int num1 = sc.nextInt();
      int num2 = sc.nextInt();
      int num3 = sc.nextInt();

      int num1num2 = (num1 + num2 + Math.abs(num1 - num2))/2;
      int num1num2num3 = (num1num2 + num3 + Math.abs(num1num2 - num3))/2;
      
      System.out.println(num1num2num3 + " eh o maior");
  }
}